lang/cs/Org.Apache.REEF.Tang/Util/ReflectionUtilities.cs

```diff
@@ -501,10 +501,10 @@ public static IEnumerable<Type> GetInterfaces(Type type, bool includeInherited)
                 return type.GetInterfaces().Except(type.BaseType.GetInterfaces());
         }
 
-        // Here is a more elaborate hack to test for annonymous type:
+        // Here is a more elaborate hack to test for anonymous type:
         // http://stackoverflow.com/questions/2483023/how-to-test-if-a-type-is-anonymous
         // compiler generated classes are always recreatable and need not additional references to check for.
-        public static bool IsAnnonymousType(Type type)
```

Comment:
Public method in non-example non-test code => needs to go through "Obsolete in 0.14" path